src/NzbDrone.Core/MediaFiles/MediaInfo/MediaInfoModel.cs

```diff
@@ -57,5 +58,8 @@ public class MediaInfoModel : IEmbeddedDocument
         public List<string> Subtitles { get; set; }
 
         public string ScanType { get; set; }
+
+        [JsonIgnore]
+        public string Title { get; set; }
```

Comment:
This is a bit generic, but not sure it's wroth going into much detail since it's not stored.